src/sdg_hub/blocks/__init__.py

```
@@ -1,6 +1,11 @@
+"""Block implementations for SDG Hub.
+
+This package provides various block implementations for data generation, processing, and transformation.
+"""
+
 # Local
 from .block import *
 from .filterblock import *
 from .iterblock import *
 from .llmblock import *
```

Pylint (3.3.7)

[error] Cannot import 'llmblock' due to 'invalid syntax (sdg_hub.blocks.llmblock, line 331)'

(E0001)